README.md

```diff
@@ -2249,6 +2250,25 @@ custom allocator can be copied from the one embedded in the builder
 library source.
 
 
+### Custom Asserts
+
```

Comment:
Suggest different wording:

On systems where the default POSIX `assert` call is unavailable, or when a different assert behaviour is desirable, it is possible to override the default behaviour via logic defined in [flatcc_assert.h](include/flatcc/flatcc_assert.h).